README.md

```diff
@@ -20,4 +20,39 @@
 </div>
 
 ## gofuzz
-**gofuzz** is a CLI tool to add bug detection capabilities into your Go code. It transforms the source code and replace calls to functions/methods of interests by calls to corresponding hooks in the `github.com/CodeIntelligenceTesting/gofuzz/sanitizers` module. **gofuzz** does not change the code in-place, but generates the instrumented source code in a temporary directory and produces an [overlay file](https://go.dev/doc/go1.16) that can be used by your build tools
\ No newline at end of file
+**gofuzz** is a CLI tool to add bug detection capabilities into your Go code. It transforms the source code and replace calls to functions/methods of interests by calls to corresponding hooks in the `github.com/CodeIntelligenceTesting/gofuzz/sanitizers` module. **gofuzz** does not change the code in-place, but generates the instrumented source code in a temporary directory and produces an [overlay file](https://go.dev/doc/go1.16) that can be used by your build tools
+
+## How to use
+1. (Once) Install the **gofuzz** CLI
+    ```shell
+    go install github.com/CodeIntelligenceTesting/gofuzz/cmd/gofuzz@latest
```

Comment:
Go 1.17 required, which should probably be mentioned somewhere in the readme. Are there other requirements?

Reply:
We require Go 1.18 as our dependencies need it. This command should work since the binary can be built with a single `go build` command. Added a mention for the required version